pkgs/os-specific/darwin/apple-source-releases/default.nix

```diff
@@ -40,6 +40,7 @@ let
       basic_cmds    = "55";
       adv_cmds      = "163";
       file_cmds     = "264.1.1";
+      shell_cmds    = "198";
```

Comment:
The version of [10.11.6](https://opensource.apple.com/release/os-x-10116.html) is shell_cmds-187, I think this one is from 10.12.1.